main.tf

```diff
@@ -92,7 +93,9 @@ module "bastion" {
   bastion_notification_protocol = var.bastion_notification_protocol
   bastion_notification_topic    = var.bastion_notification_topic
 
+  # tags
   freeform_tags = var.freeform_tags["bastion"]
+  defined_tags  = var.defined_tags["bastion"]
```

Comment:
The bastion module (https://github.com/oracle-terraform-modules/terraform-oci-bastion) doesn't have `defined_tags` yet. Please either
1. Remove it for now and add it for bastion when we've added `defined_tags` in the bastion module.
2. Or implement `defined_tags` in the bastion module. We'll cut a new release for it so we can then use this.

Reply:
I removed them for the moment